GCP automatic propagator doesn't work as per README

Following this README:
<img width="999" alt="image" src="https://user-images.githubusercontent.com/26136082/193811371-073f60f4-6457-4c95-a4d3-591888911718.png">

I have the following packages as dependencies

```
  implementation("io.opentelemetry:opentelemetry-api:1.18.0")
  implementation("com.google.cloud.opentelemetry:propagators-gcp:0.23.0-alpha")
```

I have the following jvm args:
```
"-javaagent:" + project.buildDir.toString() + "/otel/opentelemetry-javaagent.jar", "-otel.propagators=gcp")
```

And as a result I got:

```
OpenTelemetry Javaagent failed to start
io.opentelemetry.sdk.autoconfigure.spi.ConfigurationException: Unrecognized value for otel.propagators: gcp. Make sure the artifact including the propagator is on the classpath.
        at io.opentelemetry.sdk.autoconfigure.PropagatorConfiguration.getPropagator(PropagatorConfiguration.java:62)
        at io.opentelemetry.sdk.autoconfigure.PropagatorConfiguration.configurePropagators(PropagatorConfiguration.java:43)
        at io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdkBuilder.build(AutoConfiguredOpenTelemetrySdkBuilder.java:381)
        at io.opentelemetry.javaagent.tooling.OpenTelemetryInstaller.installOpenTelemetrySdk(OpenTelemetryInstaller.java:41)
        at io.opentelemetry.javaagent.tooling.AgentInstaller.installBytebuddyAgent(AgentInstaller.java:112)
        at io.opentelemetry.javaagent.tooling.AgentInstaller.installBytebuddyAgent(AgentInstaller.java:94)
        at io.opentelemetry.javaagent.tooling.AgentStarterImpl.internalStart(AgentStarterImpl.java:86)
        at io.opentelemetry.javaagent.tooling.AgentStarterImpl.start(AgentStarterImpl.java:67)
        at io.opentelemetry.javaagent.bootstrap.AgentInitializer.initialize(AgentInitializer.java:35)
        at io.opentelemetry.javaagent.OpenTelemetryAgent.startAgent(OpenTelemetryAgent.java:57)
        at io.opentelemetry.javaagent.OpenTelemetryAgent.premain(OpenTelemetryAgent.java:45)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:513)
        at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndCallPremain(InstrumentationImpl.java:525)
Error: Could not find or load main class .otel.propagators=gcp
Caused by: java.lang.ClassNotFoundException: /otel/propagators=gcp
```

--- a/propagators/gcp/README.md
+++ b/propagators/gcp/README.md
@@ -13,7 +13,11 @@ The preferred mechanism is to use the [SDK autoconfigure extension](https://gith
 
 - Add a runtime dependency from your java project to this library.
 - You can now use `oneway-gcp` and `gcp` as viable propagation strings in [the `otel.propagators` flag](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#propagator)
-
+- When using the autoinstrumentation agent, you also need to pass our propagators to the autoinstrumentation "extension" classpath. This can be done through the `otel.javaagent.experimental.extensions` flag. 
+    ```
+    -Dotel.javaagent.experimental.extensions=/path/to/downloaded/gcp-propagator
+    ```
+    For a complete example see [here](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/main/examples/autoinstrument/build.gradle#L63).
 
 ## Manual Configuration
 

--- a/propagators/gcp/README.md
+++ b/propagators/gcp/README.md
@@ -15,7 +15,7 @@ The preferred mechanism is to use the [SDK autoconfigure extension](https://gith
 - You can now use `oneway-gcp` and `gcp` as viable propagation strings in [the `otel.propagators` flag](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#propagator)
 - When using the autoinstrumentation agent, you also need to pass our propagators to the autoinstrumentation "extension" classpath. This can be done through the `otel.javaagent.experimental.extensions` flag. 
     ```
-    -Dotel.javaagent.experimental.extensions=/path/to/downloaded/gcp-propagator
+    -Dotel.javaagent.experimental.extensions=/path/to/downloaded/gcp-propagator.jar
     ```
     For a complete example see [here](https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/main/examples/autoinstrument/build.gradle#L63).
 

